src/components/ConfigureEmail.tsx

```diff
@@ -107,65 +100,72 @@ export const ConfigureEmail = (props: ConfigureEmailProps) => {
     }
   }
 
-  useEffect(() => {
-    getData()
-  }, [accessToken])
-
+  const emails = currentProfile?.emails
   return (
-    <div className="bootstrap-4-backport">
+    <div>
       {emails?.map(email => {
-        if (email === primaryEmail) {
+        if (email === primaryEmail?.email) {
           return (
-            <div key={email}>
-              <strong>{email} (Primary)</strong>
-              <button onClick={() => setEditEmail(!editEmail)}>
-                <img src={EditIcon} alt="edit icon" />
-              </button>
-            </div>
+            <Box
+              key={email}
+              sx={{ padding: '15px 0px', borderBottom: '1px solid #dcdcdc' }}
+            >
+              {email}{' '}
+              <Chip
+                sx={{ color: '#fff', marginLeft: '10px' }}
+                label="PRIMARY"
+                color="warning"
+              />
+            </Box>
           )
         } else {
           return (
-            <div key={email}>
-              {email}
+            <Box
+              key={email}
+              sx={{
+                display: 'flex',
+                padding: '15px 0px',
+                borderBottom: '1px solid #dcdcdc',
```

Comment:
if we end up having a number of places with those line separators we can use a [`<Divider/>`  ](https://mui.com/material-ui/react-divider/). We can configure margins in theme to make it easier.